css-layout-classes/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>Layout</title>
+  </head>
+
+  <body>
+
+    <div id="body">
+      <div id="title">
```

Comment:
This should be a class related to a column. So in this case, an appropriate name would be `.col-full`